parts/kuberneteswindowssetup.ps1

```diff
@@ -59,14 +50,37 @@ $global:SecurityGroupName = "{{{nsgName}}}"
 $global:VNetName = "{{{virtualNetworkName}}}"
 $global:RouteTableName = "{{{routeTableName}}}"
 $global:PrimaryAvailabilitySetName = "{{{primaryAvailablitySetName}}}"
+$global:NetworkPolicy = "{{{networkPolicy}}}"
+
+# Kubelet
+$global:KubeDir = "c:\k"
+$global:KubeletStartFile = $global:KubeDir + "\kubeletstart.ps1"
+$global:KubeProxyStartFile = $global:KubeDir + "\kubeproxystart.ps1"
+$global:KubeBinariesSASURL = "https://acsengine.blob.core.windows.net/wink8s/v1.5.3int.zip"
+$global:DockerServiceName = "docker"
+
+# CNI
+$global:CNIDir = "c:\cni"
+$global:CNIBinDir = $global:CNIDir + "\bin"
+$global:CNIConfDir = $global:CNIDir + "\netconf"
+$global:CNIKubeletOptions = " --network-plugin=cni --cni-bin-dir=$global:CNIBinDir --cni-conf-dir=$global:CNIConfDir"
+$global:CNIEnabled = $false
+
+# Transparent network
+$global:TransparentNetworkName = "transparentNet"
+$global:NatNetworkName = "nat"
+$global:PodGW = ""
+
+# Azure VNET
+$global:AzureVNetPluginURL = "https://github.com/ofiliz/hello/releases/download/pre-release/testw_v0.6.zip"
 
 filter Timestamp {"$(Get-Date -Format o): $_"}
 
 function
 Write-Log($message)
 {
     $msg = $message | Timestamp
-    Write-Output $msg
+    Write-Host $msg

```

Comment:
Why changed to Write-Host? Just for debugging? I think we want to redirect output to log file.